kolibri/logger/models.py

```diff
@@ -271,5 +271,5 @@ class ExamAttemptLog(BaseAttemptLog):
     content_id = UUIDField()
     channel_id = UUIDField()
 
-    def infer_dataset(self):
+    def infer_dataset(self, *args, **kwargs):
```

Comment:
This method is called from the base class, and it passes in arguments (well, currently just one) that the subclasses may or may not need to refer to. If we don't define args/kwargs here, it will blow up when it tries to call them.